README.md

```diff
@@ -222,6 +223,14 @@ Apps that can be used across multiple social media apps.
 
 * * *
 
+### Threads
+
+|  # |  |  |  Name | Platform | Description |
+|:--:| --- | --- | --- | --- | --- |
+| 1. | 🟥 | | [Threads Downloader](https://threadsdownloader.com) | Web | Threads Video Downloader |
```

Comment:
Optional - don't mean to hijack but might help make the section seem more substantial to warrant a section

```suggestion
| 1. | 🟥 | | [Threads Downloader](https://threadsdownloader.com) | Web | Threads Video Downloader |
| 2. | 🟩 | | [ThreadFaker](https://threadfaker.com) | Web | Fake Threads post generator |
| 3. | 🟩 | | [Threads Circles](https://threadscircles.com/) | Web | Threads interaction circle generator |
```